credits.txt

```diff
@@ -61,3 +79,188 @@ Beta Testers
   Sean Fahey
   Mauricio Gomes
   Iaz Poolar
+
+Special thanks to MCOfficer for
+creating, managing, and hosting
+the nightly builds that makes
+so much of this work possible.
```

Comment:
not sure if i deserve this special spot, so i'll put it up for discussion. be honest :)

Reply:
Well, obviously, I do. Some of us can do our own builds, but for everyone else, you are solely responsible for allowing everyone to test changes to the engine. There's a fair number of us who *can* do our own builds that simply prefer the convenience of downloading the latest nightly anyway.

Not to mention, didn't you do a whole pile of stuff setting up testing stuff on Github? And then there is James, who is incredibly useful in facilitating content creation discussion on the discord.

In my books, the nightly alone is worth a special thanks. The rest is just gravy on top of that. 

And as an aside, looking through the game credits for things like Mass Effect, people are putting their favourite catering companies in there and thanks to their pets and other stuff. That might be a tad overboard, but compared to that, your contributions are right up there as one of the Devs. Just for the record, I *don't* want us to start including catering companies and pets and whatnot in the credits, but if someone is a really valuable member of the community making a crucial tool to help us build the game... I think they should be in here somehow.

Reply:
i bow to the majority. thank you.